### ISSUE TITLE
Crashing with other mods.

### ISSUE THREAD

**Issue by AustinMReppert** [rikka0w0/MobZReborn#1]
When loading with other mods that use the cloth config api an error is thrown. I guess you just need to use it as a library with gradle.
`Exception in thread "main" java.lang.module.ResolutionException: Modules mobz and cloth_config export package me.shedaniel.cloth.clothconfig.shadowed.blue.endless.jankson to module tesseract
	at java.base/java.lang.module.Resolver.resolveFail(Resolver.java:908)
	at java.base/java.lang.module.Resolver.failTwoSuppliers(Resolver.java:822)
	at java.base/java.lang.module.Resolver.checkExportSuppliers(Resolver.java:743)
	at java.base/java.lang.module.Resolver.finish(Resolver.java:379)
	at java.base/java.lang.module.Configuration.<init>(Configuration.java:140)
	at java.base/java.lang.module.Configuration.resolveAndBind(Configuration.java:494)
	at MC-BOOTSTRAP/cpw.mods.modlauncher@9.0.7/cpw.mods.modlauncher.ModuleLayerHandler.buildLayer(ModuleLayerHandler.java:70)
	at MC-BOOTSTRAP/cpw.mods.modlauncher@9.0.7/cpw.mods.modlauncher.TransformationServicesHandler.buildTransformingClassLoader(TransformationServicesHandler.java:61)
	at MC-BOOTSTRAP/cpw.mods.modlauncher@9.0.7/cpw.mods.modlauncher.Launcher.run(Launcher.java:104)
	at MC-BOOTSTRAP/cpw.mods.modlauncher@9.0.7/cpw.mods.modlauncher.Launcher.main(Launcher.java:77)
	at MC-BOOTSTRAP/cpw.mods.modlauncher@9.0.7/cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:26)
	at MC-BOOTSTRAP/cpw.mods.modlauncher@9.0.7/cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:23)
	at cpw.mods.bootstraplauncher@0.1.17/cpw.mods.bootstraplauncher.BootstrapLauncher.main(BootstrapLauncher.java:90)`

**Comment by rikka0w0** [comment]
I didnt know that cloth config supports forge, will fix this problem asap

**Comment by rikka0w0** [comment]
Try this release:
https://github.com/rikka0w0/MobZReborn/releases/tag/3.2.0_1.17.1

Can you also help me to test if modifying the life of mobs from the setting works. Thanks.

**Comment by AustinMReppert** [comment]
The mod now loads correctly. I am not quite sure what is up with the life config. It seems to have updated one of the json config files, but not the toml one? I would think the life values would need to be in a server config, but I did not see a server config.

**Comment by rikka0w0** [comment]
I decided to move to the json instead of using Forge's toml, because Forge loads the toml __after__ it loads the entity attributes (e.g. life, attack), making it not possible to customize the life and attack value.

> I would think the life values would need to be in a server config, but I did not see a server config.
That's correct. But since I switched to json (Cloth Config), on my end it is very hard to put the json into the server config folder. If im correct, the server config folder is created by Forge and not avaliable on Fabric. Forge has 3 types of configuration file: client, common and server, the first two are placed in the config folder, the same location as in Fabric. Fabric only has one type of configuration file. Currently I dont have a better solution on this, seems like we need to tell Forge, Fabric and Cloth Config to make some change at the same time.
